### PR TITLE
LPS-174701 Remove Stream usages for module: adaptive-media-image-item-selector-impl

### PR DESCRIPTION
**Pull request by julschong**

### CODE REVIEW

**Comment by catarineaguiar** [comment]
ci:test:lima

**Comment by zhangyan0701** [comment]
No regressions found.

File: modules/apps/adaptive-media/adaptive-media-image-item-selector-impl/src/main/java/com/liferay/adaptive/media/image/item/selector/internal/FileEntryAMImageURLItemSelectorReturnTypeResolver.java [hunk 1]
```diff
@@ -76,16 +73,11 @@ public String getValue(FileEntry fileEntry, ThemeDisplay themeDisplay)
...
-		Stream<MediaQuery> mediaQueryStream = mediaQueries.stream();
+		for (MediaQuery mediaQuery :
+				_mediaQueryProvider.getMediaQueries(fileEntry)) {
 
-		mediaQueryStream.map(
-			this::_getSourceJSONObject
-		).forEach(
-			sourcesJSONArray::put
-		);
+			sourcesJSONArray.put(_getSourceJSONObject(mediaQuery));
```
Comment (dantewang) [inline modules/apps/adaptive-media/adaptive-media-image-item-selector-impl/src/main/java/com/liferay/adaptive/media/image/item/selector/internal/FileEntryAMImageURLItemSelectorReturnTypeResolver.java:79]: @julschong Please use `JSONUtil.toJSONArray` to simplify, it's like

```
		).put(
			"sources",
			JSONUtil.toJSONArray(
				_mediaQueryProvider.getMediaQueries(fileEntry),
				this::_getSourceJSONObject)
```

**Comment by dantewang** [comment]
@julschong Please resend, thanks.

File: modules/apps/adaptive-media/adaptive-media-image-item-selector-impl/src/main/java/com/liferay/adaptive/media/image/item/selector/internal/FileEntryAMImageURLItemSelectorReturnTypeResolver.java (hunk 1 above)
Comment (julschong) [reply modules/apps/adaptive-media/adaptive-media-image-item-selector-impl/src/main/java/com/liferay/adaptive/media/image/item/selector/internal/FileEntryAMImageURLItemSelectorReturnTypeResolver.java:79]: Thanks for the tip!

**Pull request closed without merging**